src/monero_key.c

```diff
@@ -379,40 +379,44 @@ int monero_apdu_get_key() {
 /* ---                                                                 --- */
 /* ----------------------------------------------------------------------- */
 int monero_apdu_verify_key() {
-    unsigned char pub[32];
-    unsigned char priv[32];
-    unsigned char computed_pub[32];
+    unsigned char pub[KEY_SIZE];
+    unsigned char priv[KEY_SIZE];
+    unsigned char computed_pub[KEY_SIZE];
     unsigned int verified = 0;
     int err = 0;
 
     err = monero_io_fetch_decrypt_key(priv, sizeof(priv));
     if (err) {
+        explicit_bzero(priv, sizeof(priv));
         return err;
     }
-    monero_io_fetch(pub, 32);
+    monero_io_fetch(pub, KEY_SIZE);
     switch (G_monero_vstate.io_p1) {
         case 0:
             err = monero_secret_key_to_public_key(computed_pub, priv, sizeof(computed_pub),
```

Comment:
You can allocate `priv` and call `monero_io_fetch_decrypt_key` only in this scope